package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "export": "next build && next export && touch out/.nojekyll && cp CNAME out/",
+    "export": "next build && next build && touch out/.nojekyll && cp CNAME out/",
```

Comment:
Hm.. why is `next build` followed by `next build` again?

Reply:
I guess, you likely wanted to remove it instead. 
<img width="1199" alt="image" src="https://github.com/geniusyield/atlas-docs/assets/29835247/4beea2c2-0f54-4bfb-a55a-1058e2251d52">

Reply:
Done:
4c0a6608eb2303d237e71423dbf3e67656123f54